TempoCore/Source/TempoCoreShared/Public/TempoCoreSettings.h

```diff
@@ -50,6 +51,11 @@ class TEMPOCORESHARED_API UTempoCoreSettings : public UDeveloperSettings
 	UPROPERTY(EditAnywhere, Config, Category="Time|FixedStep", meta=(ClampMin=1, UIMin=1, UIMax=100))
 	int32 SimulatedStepsPerSecond = 10;
 
+	// The largest time step allowed in WallClock time mode. No limit if 0.0. Note that setting this to non-zero violates 
+	// WallClock time mode's guarantee of strictly advancing along with wall clock at the step that would have exceeded the max.
+	UPROPERTY(EditAnywhere, Config, Category="Time|WallClock", meta=(ClampMin=0.0, UIMin=0.0, UIMax=1.0))
```

Comment:
Do you really want to set a `UIMax` of 1.0 here?

Reply:
Synced offline, yes the 1.0 is intentional (as a suggested limit)